sklearn/utils/estimator_checks.py

```diff
@@ -483,8 +484,61 @@ def check_dict_unchanged(name, Estimator):
                               'Estimator changes __dict__ during %s' % method)
 
 
+def _is_public(attr):
```

Comment:
renamed to `is_public_parameter`, since I use it only once and in "not inversed" version